schemas/v3.1/meta/base.schema.yaml

```diff
@@ -44,19 +48,17 @@ $defs:
         type: boolean
     type: object
     unevaluatedProperties: false
+dependentSchemas:
+  discriminator:
+    anyOf:
+    - required: [ oneOf ]
+    - required: [ anyOf ]
+    - required: [ allOf ]
 $dynamicAnchor: meta
 $id: https://spec.openapis.org/oas/3.1/meta/base
```

Comment:
I noticed that this schema's keywords are all mixed up. I know that wasn't introduced in this schema, but it would be nice to fix that while we're making changes anyway.

Reply:
I'm not sure what you mean here.

Reply:
I mean the ordering is not the same as the JSON version. Things that are best practice to put at the top of the document (`$id`, `$schema`, `$vocabularies`) are at the bottom of the document. Unorganized schemas are harder to read.